charts/redpanda/templates/statefulset.yaml

```
@@ -85,6 +85,9 @@ spec:
             runAsUser: 0
             runAsGroup: 0
           volumeMounts:
+{{- if get .Values.statefulset.initContainers.tuning "extraVolumeMounts" -}}
```

initContainers on down are not required parameters in the schema. We should either require them or use dig for each of these:
```suggestion
{{- if dig  "initContainers" "tuning" "extraVolumeMounts" false .Values.statefulset -}}
```

---

oh i see what you mean, alright.